frontend/src/components/maps/__snapshots__/MapFilterBar.test.tsx.snap

```diff
@@ -22,7 +22,7 @@ exports[`MapFilterBar renders correctly 1`] = `
             className="bar-item col"
           >
             <div
-              className="form-group"
+              className="  form-group"
```

Comment:
Why the empty space?

Reply:
I replied in a previous comment, hopefully it makes sense